pocs/camera/camera.py

```diff
@@ -707,6 +705,30 @@ def _setup_observation(self, observation, headers, filename, **kwargs):
         # be override by passed parameter so update here.
         metadata['exptime'] = exptime
 
+        # Prepare the filterwheel
+        if self.filterwheel is not None:
+
+            # Check that the filterwheel is not moving
+            if self.filterwheel.is_moving:
+
+                msg = f'Attempt to prepare observation on {self} while' \
+                        ' filterwheel is moving, ignoring.'
+                self.logger.error(msg)
+                raise error.PanError(msg)
+
+            # Move filterwheel if necessary
+            if observation.filter_name is not None:
```

Comment:
Might consider logging a message if the observation has a `filter_name`, but the camera doesn't have a filter wheel. A warning might be over the top, but perhaps `self.logger.info()`.